functions/venue.js

```diff
@@ -268,6 +268,19 @@ const removeAdmin = async (adminId) => {
     });
 };
 
+/** Builds room to be inserted
+ *
+ * @param {Room} room
+ */
+const buildRoom = (room) => {
+  const { type, ...newRoom } = room;
+  if (type) {
+    return type ? room : newRoom;
+  } else {
+    return newRoom;
+  }
```

Comment:
Why is there an `if` statement and a ternary that seem to be doing the exact same thing here?

Reply:
And what is the purpose of this function in general? The description doesn't really explain what it's doing/why, and i'm struggling to decipher the intent/need for it from the logic as well.

Reply:
Just ternary operator should work. Missed cleaning up the if construct.

The function is named `buildRoom` as it builds the `room` object. It removes the `type` attribute if it is empty. I can add a comment as to why we are doing this to make it clear for posterity.

Reply:
> It removes the type attribute if it is empty.

Fair enough, this makes sense. The naming 'buildRoom' seems strange given the intent seems to be just to remove the type attribute when empty.